pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -225,6 +228,13 @@ func NewKubeGenericRuntimeManager(
 		}
 	}
 
```

Comment:
should we check/complain if the flags are set and the feature is off? that could be confusing as to why the feature is not working

Reply:
added a check for this